src/topology/algebra/uniform_group.lean

```diff
@@ -746,3 +750,98 @@ begin
       apply h ; tauto } }
 end
 end dense_inducing
+
+section complete_quotient
+
+universe u
+open topological_space classical
+
+/-- The quotient `G ⧸ N` of a complete first countable topological group `G` by a normal subgroup
+is itself complete. [N. Bourbaki, *General Topology*, IX.3.1 Proposition 4][bourbaki1966b] -/
+@[to_additive "The quotient `G ⧸ N` of a complete first countable topological additive group
+`G` by a normal additive subgroup is itself complete. Consequently, quotients of Banach spaces by
+subspaces are complete. [N. Bourbaki, *General Topology*, IX.3.1 Proposition 4][bourbaki1966b]"]
+instance quotient_group.complete_space' (G : Type u) [group G] [topological_space G]
+  [topological_group G] [first_countable_topology G] (N : subgroup G) [N.normal]
+  [@complete_space G (topological_group.to_uniform_space G)] :
+  @complete_space (G ⧸ N) (topological_group.to_uniform_space (G ⧸ N)) :=
```

Comment:
I think it would make sense to add a note about why this `topological_group.to_uniform_space` stuff is needed here and in the next instance.